tests/ballerina-unit-test/src/test/resources/test-src/expressions/lambda/negative/fp-struct-incorrect-arg-negative.bal

```diff
@@ -1,7 +1,7 @@
 type Person record {
```

Comment:
Shall we add the  license header?

Reply:
Added with commit 6058bf3 for all missing instances